optional/config/tech-docs.yml.tt

```diff
@@ -16,3 +16,8 @@ header_links:
 
 # Tracking ID from Google Analytics (e.g. UA-XXXX-Y)
 ga_tracking_id:
+
+# Maximum heading level to include in the ToC – if your ToC
```

Comment:
'Maximum heading level' implies a 1:1 relationship between this configuration option and what h[1-6]s we pull out from the document.

Would suggest something like…

> Table of contents depth – how many levels to include in the table of contents

instead?